js/print.js

```diff
@@ -0,0 +1,316 @@
+
+  return insideCover;
+}
+
+function generateEcmaCopyrightPage() {
+  const copyrightNotice = document.createElement('div');
+
+  copyrightNotice.classList.add('copyright-notice');
+  copyrightNotice.innerHTML =
+    '<p>COPYRIGHT NOTICE</p>\n\n<p>© ' +
+    year +
+    ' Ecma International</p>\n\n<p>This document may be copied, published and distributed to others, and certain derivative works of it may be prepared, copied, published, and distributed, in whole or in part, provided that the above copyright notice and this Copyright License and Disclaimer are included on all such copies and derivative works. The only derivative works that are permissible under this Copyright License and Disclaimer are: </p>\n\n<p>(i) works which incorporate all or portion of this document for the purpose of providing commentary or explanation (such as an annotated version of the document),</p>\n\n<p>(ii) works which incorporate all or portion of this document for the purpose of incorporating features that provide accessibility,</p>\n\n<p>(iii) translations of this document into languages other than English and into different formats and</p>\n\n<p>(iv) works by making use of this specification in standard conformant products by implementing (e.g. by copy and paste wholly or partly) the functionality therein.</p>\n\n<p>However, the content of this document itself may not be modified in any way, including by removing the copyright notice or references to Ecma International, except as required to translate it into languages other than English or into a different format.</p>\n\n<p>The official version of an Ecma International document is the English language version on the Ecma International website. In the event of discrepancies between a translated version and the official version, the official version shall govern.</p>\n\n<p>The limited permissions granted above are perpetual and will not be revoked by Ecma International or its successors or assigns.</p>\n\n<p>This document and the information contained herein is provided on an &ldquo;AS IS&rdquo; basis and ECMA INTERNATIONAL DISCLAIMS ALL WARRANTIES, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO ANY WARRANTY THAT THE USE OF THE INFORMATION HEREIN WILL NOT INFRINGE ANY OWNERSHIP RIGHTS OR ANY IMPLIED WARRANTIES OF MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE.</p>';
+
+  return copyrightNotice;
+}
+
+/**
+ * A little content rearranging specifically relevant to 262
+ * */
+function ecma262fixes() {
+  const toc = document.getElementById('toc');
+
+  // eslint-disable-next-line prettier/prettier
+  specContainer.insertBefore(document.getElementById('sec-bibliography'), document.getElementById('sec-colophon'));
+  Array.from(toc.getElementsByTagName('a')).forEach(anchor => {
+    if (anchor.getAttribute('href') === '#sec-colophon') {
+      toc.getElementsByTagName('ol')[0].appendChild(anchor.parentNode);
+    }
+  });
+}
```

Comment:
I think it'd be fine to just rearrange these in the 262 spec document directly.

Reply:
I’d like very much to address that in a separate PR as well.